src/js/utils/tooltip-defaults.js

```diff
@@ -1,5 +1,7 @@
 export const defaults = {
   trigger: 'manual',
+  arrow: true,
```

Comment:
@BrianSipple this is the config change and then we just remove popper arrow stuff. @rwwagner90 I do believe that theming is affected, but perhaps easier in Tippy?

https://atomiks.github.io/tippyjs//#creating-custom-themes-creating-a-theme

Reply:
@chuckcarpenter @rwwagner90 Agreed that Tippy offers plenty of potential for better themeing. My impression is that we planning that as [part of 3.0](https://github.com/shipshapecode/shepherd/projects/4)

Reply:
agree, but I think setting this to the default arrow is definitely for now.